### ISSUE TITLE
🐞 - tui-editor-socket doesn't render content on SSR

### ISSUE THREAD

**Issue by KhaledGabr** [taiga-family/editor#835]
### Description

The following doesn't get rendered on the server. 
This is critical because search engine can't index any of the content there.

`<tui-editor-socket [content]="data"></tui-editor-socket>
`

### Angular version

17.2.3

### Taiga UI version

Taiga 3.68.0  / Tui editor 1.16.0

### Which browsers have you used?

- [X] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### Which operating systems have you used?

- [X] Windows
- [ ] macOS
- [ ] Linux
- [ ] iOS
- [ ] Android

**Comment by splincode** [comment]
try v2.0.5-alpha.0

**Comment by waterplea** [comment]
@KhaledGabr what are you providing for `TUI_EDITOR_SANITIZER`?

**Comment by splincode** [comment]
@KhaledGabr add your repo for reproduce

**Comment by waterplea** [comment]
> I wasn't aware of TUI_EDITOR_SANITIZER, but I provided that as such but doesn't seem to work:

Previously it was `TUI_SANITIZER` because we also needed it in main Taiga UI code, now it's no longer needed there so instead we added a new token `TUI_EDITOR_SANITIZER` to this package. @splincode is responsible for editor, but he is currently on vacation so this issue will have to wait for a week as the rest of the team is focused on main Taiga UI library getting ready for 4.0, sorry for the delay.

**Comment by splincode** [comment]
Fixed by `2.0.21-alpha.15`

<img width="505" alt="image" src="https://github.com/user-attachments/assets/f037bebd-ae2f-4681-8902-170fe5ea843c">
<img width="631" alt="image" src="https://github.com/user-attachments/assets/e9a17459-589b-43e1-80a9-845f7bdfbcf4">

```json
"dependencies": {
    "@angular/animations": "18.0.6",
    "@angular/cdk": "18.0.6",
    "@angular/common": "18.0.6",
    "@angular/compiler": "18.0.6",
    "@angular/core": "18.0.6",
    "@angular/forms": "18.0.6",
    "@angular/platform-browser": "18.0.6",
    "@angular/platform-browser-dynamic": "18.0.6",
    "@angular/platform-server": "18.0.6",
    "@angular/router": "18.0.6",
    "@angular/ssr": "18.0.7",
    "@commercelayer/js-auth": "5.2.1",
    "@commercelayer/sdk": "5.36.0",
    "@ctrl/tinycolor": "4.1.0",
    "@stripe/stripe-js": "4.1.0",
    "@supabase/supabase-js": "2.44.2",
    "@taiga-ui/addon-charts": "4.0.0-rc.6",
    "@taiga-ui/addon-commerce": "4.0.0-rc.6",
    "@taiga-ui/addon-doc": "4.0.0-rc.6",
    "@taiga-ui/addon-mobile": "4.0.0-rc.6",
    "@taiga-ui/addon-table": "4.0.0-rc.6",
    "@taiga-ui/cdk": "^4.0.0-rc.6",
    "@taiga-ui/core": "4.0.0-rc.6",
    "@taiga-ui/editor": "2.0.21-alpha.15",
    "@taiga-ui/event-plugins": "4.0.1",
    "@taiga-ui/experimental": "4.0.0-rc.6",
    "@taiga-ui/i18n": "4.0.0-rc.6",
    "@taiga-ui/icons": "4.0.0-rc.6",
    "@taiga-ui/kit": "4.0.0-rc.6",
    "@taiga-ui/layout": "4.0.0-rc.6",
    "@taiga-ui/legacy": "4.0.0-rc.6",
    "@taiga-ui/polymorpheus": "4.6.4",
    "@ng-web-apis/universal": "4.2.0",
    "@tiptap/extension-placeholder": "2.5.4",
    "@taiga-ui/styles": "4.0.0-rc.6",
    "@tiptap/extension-image": "2.5.4",
    "@tiptap/extension-link": "2.5.4",
    "@tiptap/extension-subscript": "2.5.4",
    "@tiptap/extension-superscript": "2.5.4",
    "@tiptap/extension-table": "2.5.4",
    "@tiptap/extension-table-cell": "2.5.4",
    "@tiptap/extension-table-header": "2.5.4",
    "@tiptap/extension-table-row": "2.5.4",
    "@tiptap/extension-task-item": "2.5.4",
    "@tiptap/extension-task-list": "2.5.4",
    "@tiptap/extension-text-align": "2.5.4",
    "@tiptap/extension-text-style": "2.5.4",
    "@tiptap/extension-underline": "2.5.4",
    "@tiptap/extension-youtube": "2.5.4",
    "ajv": "8.16.0",
    "angular-resizable-element": "7.0.2",
    "dompurify": "3.1.6",
    "embla-carousel": "8.1.6",
    "express": "4.19.2",
    "fast-json-patch": "3.1.1",
    "fuse.js": "7.0.0",
    "libphonenumber-js": "1.11.4",
    "linkifyjs": "4.1.3",
    "rxjs": "7.8.1",
    "tslib": "2.6.3",
    "uuid": "10.0.0"
  }
```
